<launch>
  <arg name="hand_pose_boxes" default="hand_cluster_point_indices_decomposer/boxes" />
  <arg name="object_boxes" default="/labeled_bounding_box_publisher_aligned/output/labeled_instance_boxes" />

  <node name="hand_interruption_checker"
        pkg="hand_motion_estimator" type="hand_interruption_checker.py"
        output="screen">
    <remap from="~input/hand_pose_boxes" to="$(arg hand_pose_boxes)" />
    <remap from="~input/object_boxes" to="$(arg object_boxes)" />
  </node>

</launch>
